Artiom/src/main/java/Circle_Area.java

```diff
@@ -0,0 +1,8 @@
+public class Circle_Area {
+    public static void main(String[] args) {
+        int radius = 4;
+        double pi = 3.14159265359;
+        double Area = pi * (radius * radius);
```

Comment:
Please follow the naming convention (local variable name - first letter lowercase)

Reply:
done